File: 백준/Silver/2581. 소수/소수.py
def isPrime(n):
    if n <= 1:
        return False
    for i in range(2, n//2+1):
        if n % i == 0:
            return False
    return True

m = int(input())
n = int(input())

sum = 0
min = 0

for i in range(m, n+1):
    if isPrime(i):
        if min == 0:
            min = i
        sum += i
        
if min == 0:
    print(-1)
else:
    print(sum)
    print(min)

File: 백준/Silver/2581. 소수/소수.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int is_prime(int n)
{
	if (n <= 1)
		return 0;

	for (int i = 2; i < n; i++)
	{
		if (n % i == 0)
			return 0;
	}

	return 1;
}

int main(void)
{
	int m, n, min = 0, sum = 0;
	scanf("%d", &m);
	scanf("%d", &n);

	for (int i = m; i <= n; i++)
	{
		if (is_prime(i)) 
		{
			if (min == 0)
				min = i;
			sum += i;
		}		
	}

	if (min == 0)
		printf("-1\n");
	else
		printf("%d\n%d\n", sum, min);
}